.github/workflows/backport.yml

```diff
@@ -0,0 +1,43 @@
+name: Backport
+on:
+  workflow_dispatch:
+    inputs:
+      number:
+        description: "The pull request # to backport"
+        required: true
+
+jobs:
+  backport:
+    runs-on: ubuntu-latest
+    steps:
+      - run: |
+          if [[ ! $GITHUB_REF_NAME =~ ^release/v[0-9]+\.[0-9]+\.x-0\.[0-9]+bx$ ]]; then
```

Comment:
For some reason, this regex isn't picking up my release branch named `release/v1.13.0-0.33b0`. Does the "long term branch" have to have "x" or is that a wildcard of some sort?

Reply:
did you manually create the `release/v1.13.0-0.33b0` branch? the prepare-release-branch workflow should have named it `release/v1.13.x-0.33bx`, e.g. https://github.com/trask/opentelemetry-python/tree/release/v1.12.x-0.33bx

the naming is to indicate that the release branch covers all patch releases (e.g. v1.12.1-0.33b1 will be released from that same release branch)

Reply:
You're right, it works as expected now.